src/app/app.module.ts

```diff
@@ -31,7 +30,7 @@ import { DishService } from './services/dish.service';
     MatCardModule,
     MatButtonModule
   ],
-  providers: [DishService],
+  providers: [],
```

Comment:
why have we remove service from providers, how is it provided now?

Reply:
It's provided via "providedIn" property
https://angular.io/guide/singleton-services